lib/triangles.js

```diff
@@ -0,0 +1,22 @@
+function isTriangle(sideOne, sideTwo, sideThree){
```

Comment:
there's a LOT of conditionals here, could we break this down and make it more single responsibility?